command/acl_token_update.go

```diff
@@ -38,6 +41,12 @@ Update Options:
   -policy=""
     Specifies a policy to associate with the token. Can be specified multiple times,
     but only with client type tokens.
+
+  -role-id=""
+     ID of a role to use for this token. May be specified multiple times.
+
+  -role-name=""
+     Name of a role to use for this token. May be specified multiple times.
```

Comment:
```suggestion
  -role-id=""
    ID of a role to use for this token. Can be specified multiple times, but
    only with client type tokens.

  -role-name=""
    Name of a role to use for this token. Can be specified multiple times, but
    only with client type tokens.
```

Match the help message from `-policy`. I'm pretty sure the client type restriction also applies.